src/main/java/org/apache/bcel/classfile/ConstantUtf8.java

```diff
@@ -88,39 +128,11 @@ public void run() {
         }
     }
 
-    /**
```

Comment:
You cannot remove existing public methods. That causes a break in binary compatibility.

Reply:
Let me fix that